.github/sync.yml

```diff
@@ -0,0 +1,3 @@
+celestiaorg/celestia-core:
+  - source: pkg/consts/consts.go
+    dest: payment/consts/celestia_core_consts.go
```

Comment:
still need to try this with a github token, but this essentially attempts to copy over the celestia-core/pkg/consts package every day, if there are any differences, it opens a PR